### DISCUSSION TITLE
请求加入回复列表自动展开功能

### DISCUSSION THREAD

**Discussion by monkeycathyd** [the1812/Bilibili-Evolved#4797] [功能建议 / Ideas]
有时回复列表会像图中所示显示不完全或完全隐藏，甚至有时会过滤掉一些正常的楼中楼，需要另外点击才能展开。希望可以自动展开这些楼层
![屏幕截图 2024-06-23 222847](https://github.com/the1812/Bilibili-Evolved/assets/103822978/5a0bd46b-8c14-4ed1-83e8-1d77b3641b2b)

**Comment by Geeyun-JY3** [comment]
请善用搜索，重复：#4286
